src/main/components/styled-components/Clock/Clock.brs

```diff
@@ -22,6 +25,18 @@ sub init()
     updateClock()
 end sub
 
+'Update font
```

Comment:
These methods looks great! We're trying to use [brightscript-doc](https://developer.roku.com/en-ca/docs/developer-program/dev-tools/brightscript-doc.md) comments (similar to javadoc), so if you could add an `@param` line to each of these, that would be awesome.